<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviorForm</class>
 <widget class="QWidget" name="BehaviorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Behavior</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="loadUshortBox">
     <property name="toolTip">
      <string>Store images in program memory as 16-bit integers instead of converting to bytes</string>
     </property>
     <property name="text">
      <string>Load non-byte data into program as 16-bit integers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="startAtMidZBox">
     <property name="toolTip">
      <string>Start program on middle section instead of first one</string>
     </property>
     <property name="text">
      <string>Show middle section when program starts</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1_2">
       <property name="text">
        <string>Do autocontrast at start</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="autoConAtStartBox">
       <property name="toolTip">
        <string>Set contrast with &quot;Auto&quot; on first section shown</string>
       </property>
       <item>
        <property name="text">
         <string>Never</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>If no model loaded</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Always</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="spacer23">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="geomCheckBox">
     <property name="text">
      <string>Save &amp;&amp; restore position of Zap &amp;&amp; Info windows</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="startInHQBox">
     <property name="text">
      <string>Start Zap &amp;&amp; XYZ windows with high-quality display</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="classicBox">
     <property name="text">
      <string>Start Slicer in classic mode (no panning)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="silenceBox">
     <property name="toolTip">
      <string>Turn off the beep for error mesages</string>
     </property>
     <property name="text">
      <string>Silence alarms (beeps)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="selectONcheckBox">
     <property name="toolTip">
      <string>Ignore objects that are OFF when attaching with mouse</string>
     </property>
     <property name="text">
      <string>First mouse button selects points only in visible objects</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="slicerNewSurfBox">
     <property name="toolTip">
      <string>When modeling closed contours in slicer, start a new surface when the angle changes</string>
     </property>
     <property name="text">
      <string>Start new surfaces upon angle change in Slicer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="arrowsScrollZapBox">
     <property name="text">
      <string>Up/Down arrows act like Page Up/Down in Zap</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Step size for big Page Up/Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pageStepSpinBox">
       <property name="toolTip">
        <string>Step size for paging up and down with keypad * and / hot keys</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer23_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel3">
       <property name="text">
        <string>Step size for F1-F8 hotkeys:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="f1f8StepSpinBox">
       <property name="toolTip">
        <string>Step change in black/white values with F1 to F8 keys</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Spacer8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox5">
     <property name="title">
      <string>Model autosave control</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="autosaveEnabledBox">
          <property name="toolTip">
           <string>Save model to #autosave# file periodically</string>
          </property>
          <property name="text">
           <string>Autosave enabled</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel7">
          <property name="text">
           <string>Interval (min):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="autosaveSpinBox">
          <property name="toolTip">
           <string>Enter time in minutes between model autosaves</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabel8">
          <property name="text">
           <string>Directory:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="autosaveDirEdit">
          <property name="toolTip">
           <string>Enter a directory for autosave files to be placed in</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox3">
     <property name="title">
      <string>Behavior when main window is parked</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="imageIconifyBox">
        <property name="toolTip">
         <string>Park all image windows (Zap, Slicer, etc) when parking Info Window</string>
        </property>
        <property name="text">
         <string>Image windows iconify with Info Window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="imodDlgIconifyBox">
        <property name="toolTip">
         <string>Park all 3dmod dialogs (Edit-Object-Type, etc) when parking Info Window</string>
        </property>
        <property name="text">
         <string>3dmod dialogs iconify with Info Window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="imodvDlgIconifyBox">
        <property name="toolTip">
         <string>Park all 3dmodv dialogs (Edit-Objects, etc)  when parking 3dmodv Window</string>
        </property>
        <property name="text">
         <string>3dmodv dialogs iconify with Model View Window</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer14">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
